querqy-for-lucene/querqy-lucene/src/test/java/querqy/lucene/rewrite/LuceneQueryBuilderTest.java

```diff
@@ -332,9 +348,10 @@ public void test13() throws Exception {
    }
 
     @Test
-    public void testMultiMatch01() throws Exception {
+    public void testMultiMatchNotApplied() throws Exception {
```

Comment:
I am a big fan of having something like MultiMatchDismaxQueryStructurePostProcessorTest.class